hr_timesheet_sheet_no_negative/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============================
+HR Timesheet Sheet No Negative
+==============================
+
+This module disable negative hours.
```

Comment:
I think you should clarify here that it's a functionality regarding the filling of timesheet sheet lines.